api_app/db/migrations/shared_services.py

```diff
@@ -15,8 +15,8 @@ def deleteDuplicatedSharedServices(self) -> bool:
         for template_name in template_names:
             for item in self.query(query=f'SELECT * FROM c WHERE c.resourceType = "shared-service" AND c.templateName = "{template_name}" \
                                                                 ORDER BY c.updatedWhen ASC OFFSET 1 LIMIT 10000'):
-                logging.INFO(f"Deleting element {item}")
-                self.delete_item(item, partition_key=item["id"])
```

Comment:
This must've been a leftover from some old version of the library used? I remember it required for this parameter to be passed through...